### ISSUE TITLE
[RFC] Quick access to WhatsApp Web in the phone widget.

### ISSUE THREAD

**Issue by adasatorres** [OCA/web#3011]
Good morning, I would like to propose a development that modifies the behavior of the phone widget to add a button that allows opening a conversation with the number entered in the field using WhatsApp Web. Below, I outline several reasons why I believe this development would be useful:

1. **Accessibility**: Allowing users to utilize WhatsApp Web to initiate conversations without requiring WhatsApp Business expands access to communication tools, especially for small businesses or individuals who do not want or cannot afford the Business version.
2. **Cost Savings**: By offering a practical functionality without requiring subscriptions, this helps users looking for cost-effective solutions to manage their communication.
3. **Convenience**: Simplifying the process to open a conversation directly in WhatsApp Web from a widget can enhance the user experience, especially in environments where WhatsApp is a primary communication channel.
4. **Potential Demand**: Given that WhatsApp is widely used in many regions, a feature like this could be well-received by individuals and businesses seeking to make the most of the platform.

**Comment by rrebollo** [comment]
In general, I’m in favor of this proposal—it’s a great idea. However, I would suggest considering making this behavior an extension rather than a core feature. This approach could provide more flexibility and allow for similar integrations, such as one for Telegram or other platforms, to be implemented easily. By keeping it modular, we could cater to different use cases without overloading the core functionalities. 

What do you think about this approach?

**Comment by adasatorres** [comment]
@rrebollo  Yes, I agree.

**Comment by rrebollo** [comment]
@adasatorres You should be aware of maybe collisions with https://github.com/OCA/social/tree/fb34dbba901b0537d6e772f234e6f966eeea83cf/mail_gateway_whatsapp.

**Comment by adasatorres** [comment]
@rrebollo Yes, I will keep that in mind, although I don't think there will be any conflicts.

**Comment by jelenapoblet** [comment]
I like the idea, it makes sense to explore a 'free' solution for WhatsApp communications, as using the API from Meta Business Suite incurs additional costs.

**Comment by pedrobaeza** [comment]
But this is not a real integration. It's just a redirection/shortcut. As the answers won't be noted on Odoo, no log on Odoo, and you have to check on the WhatsApp web page if there's something new. It's OK to do it, but just take into account that the usefulness is very limited.

**Comment by adasatorres** [comment]
@pedrobaeza  apologies for the confusion; I will change the title. And yes, the utility is quite limited, as it will only serve as quick access to a conversation on WhatsApp Web or the WhatsApp app installed on the device. No information about the conversation, etc., is generated in Odoo.

**Comment by adasatorres** [comment]
Hello, here is the reference to the PR. #3030
